src/Microsoft.AspNet.Mvc.Core/ModelBinding/ByteArrayModelBinder.cs

```diff
@@ -39,7 +39,12 @@ public class ByteArrayModelBinder : IModelBinder
             try
             {
                 var model = Convert.FromBase64String(value);
-                return new ModelBindingResult(model, bindingContext.ModelName, isModelSet: true);
+
+                // We do not need to set an explict ModelValidationNode since CompositeModelBinder does that automatically.
```

Comment:
Are you going to get rid of this `CompositeModelBinder` behaviour immediately after checking this PR and the two others that depend on it in?  Or is this PR still in flux?